Userland/Libraries/LibCore/System.cpp

```diff
@@ -118,6 +118,8 @@ ErrorOr<void> sigaction(int signal, struct sigaction const* action, struct sigac
 
 #ifdef __APPLE__
 ErrorOr<sig_t> signal(int signal, sig_t handler)
+#elif defined(__OpenBSD__)
```

Comment:
We should avoid re-declaring the same signature as above.. how about instead we just modify the above `#ifdef` to something like: 

```cpp
#if defined(__APPLE__) || defined(__OpenBSD__)
```

Same goes for the header as well?